feature-staking-impl/src/main/java/jp/co/soramitsu/feature_staking_impl/presentation/validators/details/view/ValidatorInfoView.kt

```diff
@@ -44,8 +44,8 @@ class ValidatorInfoView @JvmOverloads constructor(
         validatorTotalStakeView.setExtraOrHide(fiat)
     }
 
-    fun setNominatorsCount(count: Int, maxNominations: Int) {
-        validatorNominatorsView.setBody(context.getString(R.string.staking_max_format, count, maxNominations))
+    fun setNominatorsCount(count: Int, maxNominations: Int?) {
+        validatorNominatorsView.setBody(if (maxNominations == null) count.toString() else context.getString(R.string.staking_max_format, count, maxNominations))
```

Comment:
I think we should format count here with our number formatter to prevent plain display of big numbers for example)

Reply:
Done